nav2_pkgs.txt

```diff
@@ -0,0 +1,7 @@
+dwb_msgs
```

Comment:
Why do we need this file? Maybe we can put it all together in minimal_repos.txt??

Reply:
Merging into feat/update_humble sounds good! 
Regarding the file, that was my first intention the problem is that this file is used in two commands:
Here in the rosdep install:
```
 apt-get update && rosdep install -q -y \
  --from-paths $(awk '$0="src/navigation2/"$0' ./minimal_repos.txt) \ 
  --ignore-src \
  && rm -rf /var/lib/apt/lists/*
```

And in the colcon build
```colcon build \
  --symlink-install \
  --mixin \
  $OVERLAY_MIXINS \
  --packages-select $(cat ./nav2_pkgs.txt) $(cat ./minimal_repos.txt) \```

The problem is that in the first command it looks for packages at path : _src/navigation2_ while these packages are located in _src/navigation2/nav2_dwb_controller_ which makes this command to fail. And if you add the entire path in the minimal_repos.txt then it fails the colcon build because it expects only the name of the package so _dwb_core_ for example and not the entire path. I solved this issue by adding all these pkgs in a second file, but if you have a better alternative I am open to discuss it.

Reply:
The branch for humble is the _feat/humble_main_ no? Because I cannot find _feat/update_humble_

Reply:
Yes sorry, I meant `feat/humble_main`

I understand now the issue with the files. Lets leave it like this and if we think of a cleaner solution we can still do it in the PR of Humble. Thanks for the clarification Bianca!